src/mgmt/srv.c

```diff
@@ -60,22 +61,87 @@ static struct daos_rpc_handler mgmt_handlers[] = {
 #undef X
 
 static void
-mgmt_drpc_handler(Drpc__Call *request, Drpc__Response **response)
+process_daos_request(Proto__DaosRank *rank, Proto__DaosResponse *response)
```

Comment:
Not really a change request, just a question: Does the method name still make sense? Since it looks like it's processing a kill rank request specifically.